src/jvm/main/org/jetbrains/kotlinx/lincheck/IdeaPlugin.kt

```diff
@@ -112,10 +119,11 @@ fun beforeEvent(eventId: Int, type: String) {
  */
 @Suppress("UNUSED_PARAMETER")
 fun visualizeInstance(
-    testInstance: Any,
-    numbersArrayMap: Array<Any>,
+    testInstance: Any?,
+    objectToNumberMap: Array<Any>,
     continuationToLincheckThreadIdMap: Array<Any>,
-    threadToLincheckThreadIdMap: Array<Any>
+    threadToLincheckThreadIdMap: Array<Any>,
+    // TODO: add thread names array parameter
```

Comment:
Please create an issue for this

Reply:
@avpotapov00 should we pass thread names at all? WDYT?